app/gateway-oss/2.5.x/admin-api.md

```diff
@@ -1172,6 +1255,11 @@ Services can be both [tagged and filtered by tags](#tags).
 
 ### Add Service
 
+
```

Comment:
We have changed how we add admonitions recently. See the [Admonitions](https://docs.konghq.com/contributing/markdown-rules/#admonitions) section in the Markdown rules page of our Contribution guidelines.